cluster/terraform_kubernetes/variables.tf

```diff
@@ -184,6 +184,10 @@ variable "alertable_apps" {
   description = "Map of deployments which we want to monitor. Each key contains a map to override the default values."
   default     = {}
 }
+variable "alertmanager_external_url" {
```

Comment:
Can we use the cluster_data module ingress_domain output instead? https://github.com/DFE-Digital/terraform-modules/blob/main/aks/cluster_data/outputs.tf#L29
So we don't need the input variable

Reply:
used cluster_data module ingress_domain